python/persistenceClustering1.py

```diff
@@ -0,0 +1,124 @@
+tableToPoints1.XColumn = "X"
+tableToPoints1.YColumn = "Y"
+tableToPoints1.ZColumn = "X"
+tableToPoints1.a2DPoints = 1
+tableToPoints1.KeepAllDataArrays = 1
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints2 = TTKIcospheresFromPoints(Input=tableToPoints1)
+tTKIcospheresFromPoints2.Radius = 0.05
+
+# create a new 'Gaussian Resampling'
+gaussianResampling1 = GaussianResampling(Input=tableToPoints1)
+gaussianResampling1.ResampleField = ["POINTS", "ignore arrays"]
+gaussianResampling1.ResamplingGrid = [256, 256, 3]
+gaussianResampling1.SplatAccumulationMode = "Sum"
+
+# create a new 'Slice'
+slice1 = Slice(Input=gaussianResampling1)
+slice1.SliceType = "Plane"
+slice1.HyperTreeGridSlicer = "Plane"
+slice1.SliceOffsetValues = [0.0]
+
+# init the 'Plane' selected for 'SliceType'
+slice1.SliceType.Origin = [-0.0187499550000001, 0.0313220950000002, 0.0]
+slice1.SliceType.Normal = [0.0, 0.0, 1.0]
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=slice1)
+tTKPersistenceDiagram1.ScalarField = ["POINTS", "SplatterValues"]
+tTKPersistenceDiagram1.InputOffsetField = [None, ""]
+
+# create a new 'Threshold'
+threshold12 = Threshold(Input=tTKPersistenceDiagram1)
+threshold12.Scalars = ["CELLS", "PairIdentifier"]
+threshold12.ThresholdRange = [-1.0, -0.1]
+
+# create a new 'Threshold'
+threshold1 = Threshold(Input=tTKPersistenceDiagram1)
+threshold1.Scalars = ["CELLS", "PairIdentifier"]
+threshold1.ThresholdRange = [-0.1, 999.0]
+
+# create a new 'Extract Surface'
+extractSurface7 = ExtractSurface(Input=threshold12)
+
+# create a new 'Tube'
+tube7 = Tube(Input=extractSurface7)
+tube7.Scalars = ["POINTS", "CriticalType"]
+tube7.Vectors = ["POINTS", "Coordinates"]
+tube7.Radius = 0.75
+
+# create a new 'Threshold'
+persistenceThreshold0 = Threshold(Input=threshold1)
+persistenceThreshold0.Scalars = ["CELLS", "Persistence"]
+persistenceThreshold0.ThresholdRange = [10.0, 9999.0]
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints11 = TTKIcospheresFromPoints(Input=persistenceThreshold0)
+
+# create a new 'Extract Surface'
+extractSurface6 = ExtractSurface(Input=persistenceThreshold0)
+
+# create a new 'Tube'
+tube6 = Tube(Input=extractSurface6)
+tube6.Scalars = ["POINTS", "CriticalType"]
+tube6.Vectors = ["POINTS", "Coordinates"]
+tube6.Radius = 0.75
+
+# create a new 'TTK TopologicalSimplification'
+tTKTopologicalSimplification1 = TTKTopologicalSimplification(
+    Domain=slice1, Constraints=persistenceThreshold0
+)
+tTKTopologicalSimplification1.ScalarField = ["POINTS", "SplatterValues"]
+tTKTopologicalSimplification1.InputOffsetField = [None, ""]
+tTKTopologicalSimplification1.VertexIdentifierField = [None, ""]
+
+# create a new 'TTK MorseSmaleComplex'
+tTKMorseSmaleComplex1 = TTKMorseSmaleComplex(Input=tTKTopologicalSimplification1)
+tTKMorseSmaleComplex1.ScalarField = ["POINTS", "SplatterValues"]
+tTKMorseSmaleComplex1.OffsetField = ["POINTS", "SplatterValues"]
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.